pkgs/development/node-packages/node-env.nix

```diff
@@ -494,11 +494,13 @@ let
       extraArgs = removeAttrs args [ "name" "dependencies" "buildInputs" "dontStrip" "dontNpmInstall" "preRebuild" "unpackPhase" "buildPhase" "meta" ];
     in
     stdenv.mkDerivation ({
-      name = "${name}${if version == null then "" else "-${version}"}";
-      buildInputs = [ tarWrapper python nodejs ]
+      name = name + lib.optionalString (version != null) "-${version}";
+      strictDeps = true;
+      buildInputs = [ tarWrapper ]
         ++ lib.optional (stdenv.hostPlatform.isLinux) pkgs.util-linux
         ++ lib.optional (stdenv.hostPlatform.isDarwin) libtool
         ++ buildInputs;
+      propagatedNativeBuildInputs = [ nodejs python ];
```

Comment:
Do we really must propagate here?

Reply:
Technically yes, but it causes more fallout. I'm at 11 new failures after 30 minutes of `nixpkgs-review`. I don't want to make this PR any bigger, but I can do a follow-up PR.